cypress/page/products-list.page.ts

```diff
@@ -5,7 +5,7 @@ class ProductListPage {
   private CheckoutButton: string;
 
   constructor() {
-    this.CartSelect = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
+    this.CartSelect = ".ajax_add_to_cart_button > span";
     this.ModalView = "#layer_cart";
     this.CheckoutButton = ".clearfix div.button-container > a";
```

Comment:
`a[title="Proceed to checkout"]` should work better for this

![image](https://user-images.githubusercontent.com/10209180/167319054-79115ae5-aae1-4cd4-a643-65ad5fd25a88.png)